backend/tracim_backend/lib/core/userconfig.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any
+from typing import Dict
+
+from tracim_backend.models.auth import User
+from tracim_backend.models.tracim_session import TracimSession
+from tracim_backend.models.userconfig import UserConfig
+
+
+class UserConfigApi:
+    """Api to query user configuration"""
+
+    def __init__(self, user: User, session: TracimSession):
```

Comment:
Other Api classes name the `user` parameter as `current_user`, so I would propose to rename it for coherency:
```suggestion
    def __init__(self, current_user: User, session: TracimSession):
```